priority-queue/Readme.md

```diff
@@ -1,100 +1,129 @@
+   ```
+1. Add a filter to the low priority subscription.
 
-	* Now you need to configure the managed identities role assignments:
+   ```azurecli
+   az servicebus topic subscription rule create --name priorityFilter --namespace-name $SERVICE_BUS_NAMESPACE_NAME --resource-group $RESOURCE_GROUP_NAME --topic-name $SERVICE_BUS_TOPIC_NAME --subscription-name $SERVICE_BUS_SUBSCRIPTION_NAME_LOW --filter-sql-expression "Priority = 'lowpriority'"
+   ```
+1. Retrieve the primary connection string for the Service Bus namespace.  Copy and retain this value as it will be needed in later steps.
 
-			In the Azure portal, navigate to the Azure Service Bus Namespace that was provisioned in the first step.
-			Select Access Control (IAM). This is where you can view and configure who has access to the resource.
-			Click Add and select add role assignment.
-			From the list, select "Azure Service Bus Data Sender", click Next
-			In "Assign access to", radio button list, select "Managed identity"
-			Click on "Select members", the "Select Manage identities" dialog will show up
-			In the Managed Identity  dropdown list select "Function App"
-			Find the PriorityQueueSender function app click on it
-			Click "Select"
-			On the main dialog, click "Review + Assign"
+   ```azurecli
+   SERVICE_BUS_CONNECTION_STRING=$(az servicebus namespace authorization-rule keys list --resource-group $RESOURCE_GROUP_NAME --namespace-name $SERVICE_BUS_NAMESPACE_NAME --name RootManageSharedAccessKey --query primaryConnectionString --output tsv)
+   echo $SERVICE_BUS_CONNECTION_STRING
+   ```
 
-			For the Azure Consumer Function Apps, repeat the process but in this case use the role
-			"Azure Service Bus Data Reader"
 
-	* Once the Functions are deployed you can configure monitoring by following these steps:
+## Run the example locally
 
-		- From the Azure Portal, go to the Function App Service
-		- Click on "Functions" and select the azure function
-		- From the "developer" left menu click on "Monitor"
-		- Turn on application insights
-		- Refresh the screen
-		- Once refreshed you will see two tabs, "invocation" and "logs"
-		- From the invocations tab you can see the twenty most recent function invocation traces. For more advanced analysis, run the query in Application Insights. 
-		- From the logs tab you can see the logging information that your functions are sending.
\ No newline at end of file
+
+1. Open the `priority-queue.sln` file in Visual Studio.
+  
+1. Edit the `local.settings.json` file for each project set the value of `ServiceBusConnectionString` to the **SERVICE_BUS_CONNECTION_STRING** retained from earlier.
+
+1. Set the `PriorityQueueSender` project as startup. 
+1. Press F5 in Visual Studio to start running the example. The function will begin sending messages to the topic every 30 seconds.
+1. Stop the execution, change the startup project to either the `PriorityQueueConsumerHigh` or `PriorityQueueConsumerLow`.
```

Comment:
@skabou - what did we say next steps were about this?

Reply:
I have a new version in the works  (deploys via bicep file) - Trying to get the PR in before the end of the month.  My concern is the example is still "more or less the same" as to what is there - Essentially two pub subs, one for high priority, and one for low priority.  If we don't think publishing a demo of that is worthwhile then I could scrap the code portion.

Reply:
If it's an implementation of the pattern as we have it documented/described, I think it will be useful.  If you don't feel though it's an implementation of the pattern, then we need to re-evaluate for sure. I think if your new version does a deploy to azure for sure, then we can show the pattern very clearly with the scale-out settings.

Reply:
I do think it's still worthwhile to demonstrate.